library/marathon.py

```diff
@@ -231,21 +235,24 @@ def main():
 
     marathon_uri = module.params['uri'].rstrip('/')
     state = module.params['state']
-    json_filename = module.params['app_json']
-    app_json = ''
-    app_id = ''
```

Comment:
ps: I'ld have preferred app_id (vs appid), also for consistency with app_json var